src/es/hopscotch.js

```diff
@@ -87,12 +87,15 @@ import * as Utils from './modules/utils.js';
     getState() {
     },
     configure(configHash) {
-      if(!configHash) {
+      if (!configHash) {
         return;
       }
-      for(let prop in configHash){
+      for (let prop in configHash) {
         globalConfig.set(prop, configHash[prop]);
       }
+    },
+    resetDefaultOptions() {
+      globalConfig = new Config({}, defaultConfig);
```

Comment:
Just noticed looking over the current API... this method (and I think a few others) should return `this` for chaining. I'll file a separate issue for this.